streams/src/main/java/org/apache/kafka/streams/state/internals/AbstractMergedSortedCacheStoreIterator.java

```diff
@@ -192,4 +192,3 @@ public void close() {
         storeIterator.close();
     }
 }
-
```

Comment:
Is this intentional? We usually have a newline at file end in case some specific IDEs do not like otherwise.

Reply:
This looks strange. I can see there is a newline in my IDE. Seems that in trunk it had 2 newlines at the end. I can roll-back this change if needed.